src/ghdldrv/ghdllocal.adb

```diff
@@ -367,6 +367,8 @@ package body Ghdllocal is
             return "v93";
          when Vhdl_08 =>
             return "v08";
+         when Vhdl_19 =>
```

Comment:
I'm not sure how exactly this library compatibility concept works. Should -2019 be compatible with -2008? Is this something that can still easily be decided later?

Reply:
Yes, we can decide later.  But I think we need to have dedicated 2019 libraries, as at least `std` was extended.